web/src/App.jsx

```diff
@@ -145,6 +141,9 @@ export const useStatusStore = create((set, get) => ({
                 set({ disconnected: true });
             });
     },
+    setSystemState: (s) => {
+        set({ status: s, loaded: true, disconnected: false });
+    },
```

Comment:
I briefly included `skipUpdate: true` in this list of things, but I haven't been able to find a case where that's necessary in testing